internal/engine/upper.go

```diff
@@ -313,6 +313,8 @@ func (u Upper) handleCompletedBuild(ctx context.Context, engineState *store.Engi
 		ms.LastBuild = store.NewBuildState(cb.Result)
 		ms.LastSuccessfulDeployEdits = ms.CurrentlyBuildingFileChanges
 		ms.CurrentlyBuildingFileChanges = nil
+
+		ms.Pod.OldRestarts = ms.Pod.TotalRestarts // # of pod restarts from old code (shouldn't be reflected in HUD)
```

Comment:
so due to a bug, currently we can't container update a previously crashed pod, so this isn't relevant, but theoretically if Pod X had bad code and restarted 5 times, then we update the pod in place with new code, the pod will still think it has 5 restarts, but current code caused none of them, so we want to show the user 0 restarts.